packages/template/src/lib/stack-app/apps/implementations/client-app-impl.ts

```
@@ -1283,13 +1313,13 @@
       return this._redirectMethod.useNavigate();
     } else if (this._redirectMethod === "window") {
       return (to: string) => window.location.assign(to);
-    // IF_PLATFORM next
+      // IF_PLATFORM next
     } else if (this._redirectMethod === "nextjs") {
       const router = NextNavigation.useRouter();
```

Biome (2.1.2)

[error] This hook is being called conditionally, but all hooks must be called in the exact same order in every component render.

For React to preserve state between calls, hooks needs to be called unconditionally and always in the same order.
See https://reactjs.org/docs/hooks-rules.html#only-call-hooks-at-the-top-level

(lint/correctness/useHookAtTopLevel)